packages/gatsby-telemetry/src/error-helpers.ts

```diff
@@ -1,8 +1,17 @@
-const { sep } = require(`path`)
+import { sep } from "path"
+
+export interface IErrorWithStdErrAndStdOut extends Error {
```

Comment:
@ascorbic I'm not sure of this type but it's only used internally so it should be easy to fiddle once we type the events etc.